example/lib/main.dart

```diff
@@ -139,7 +139,9 @@ class _MyAppState extends State<MyApp> {
         appearanceMode: appearanceModeDefault,
         androidDefaultThemeResource: 'PSPDFKit.Theme.Example',
         iOSRightBarButtonItems:['thumbnailsButtonItem', 'activityButtonItem', 'searchButtonItem', 'annotationButtonItem'],
-        iOSLeftBarButtonItems:['settingsButtonItem']
+        iOSLeftBarButtonItems:['settingsButtonItem'],
+		iOSAllowToolbarTitleChange: false,
```

Comment:
Weird indendation here.

Reply:
Thank you! Addressed in https://github.com/PSPDFKit/pspdfkit-flutter/pull/22/commits/b603a7e21872db054149ab44b43fbda33cbd7cb5